perl-package/AI-MXNet-Gluon-ModelZoo/examples/image_classification.pl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env perl
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+use strict;
+use warnings;
+use AI::MXNet::Gluon::ModelZoo 'get_model';
+use AI::MXNet::Gluon::Utils 'download';
+use Getopt::Long qw(HelpMessage);
+
+GetOptions(
+    ## my Pembroke Welsh Corgi Kyuubi, enjoing Solar eclipse of August 21, 2017
+    'image=s' => \(my $image = 'https://scontent-sea1-1.cdninstagram.com/vp/'.
```

Comment:
😆 
How long does this pic persist on the CDN of instagram? Will the link expire?

Reply:
@szha I am not sure.
But I'll be glad if I can put this image on mxnet data server. Besides that I have a few blog posts that I want in future to have added as a tutorials for perl api
http://blogs.perl.org/users/sergey_kolychev/, they do contain some images and data files that would be nice to have on mxnet data server as well.
I'll be writing new blog post soon announcing ModelZoo for perl and I'll use images from https://www.facebook.com/sergey.kolychev.39/posts/10155737425303434.
Can you help me with putting these images/data files to the server ?
Or may be I can get a write access somehow so I'll not bother you with this ?

Reply:
http://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/dataset/kyuubi.jpg

Reply:
Thanks!